bindings/python/examples/client/custom_plugin.py

```diff
@@ -1,9 +1,11 @@
-from iota_sdk import Client, init_logger
-from dotenv import load_dotenv
-
 import json
 import os
 
+from dotenv import load_dotenv
+
+# pylint: disable=no-name-in-module
```

Comment:
yea these should all go, its aparently requiring a __init__.py file somewhere

Reply:
sorry I forgot to explain it in the PR. This must be disabled because `init_logger`  comes from external bindings and `pylint` doesn't work well with the case when we have a mixin of  C  and native ones important from one enrypoint (`iota_sdk`) So far I didn't find any solution that would fix that error (it's a known problem of `pylint`) I propose spend additional hours specifically on that in the separate story

Reply:
Can we define `init_logger` in the Python code which just calls the external one, so we don't need an extra comment here?

Reply:
Ok, unfortunately, it's not easy. When trying this I always ended up in an import loop. The 'right' way it should be done is with `pyi` file that contains the definitions for FFI functions. Apparently `pylint` version 2.5.17 doesn't work well those files, so we probably need to wait for version 3. So my proposition is to keep it as is